tools/cockpit.spec

```diff
@@ -352,7 +349,6 @@ Requires: shadow-utils
 Requires: grep
 Requires: libpwquality
 Requires: /usr/bin/date
-Provides: cockpit-realmd = %{version}-%{release}
```

Comment:
This is an independent change, and I think it's ok to leave this for now.

Reply:
Yes, I was wondering about that.  My reasoning for removing the `Provides` is that installing cockpit-realmd probably is meant to give you `/usr/share/cockpit/realmd`, but this this directory no longer exists...

Reply:
Right, that was a thinko of mine yesterday. With this PR, one can't install cockpit-realmd independently any more (conceptually).

Reply:
Removed this change.

Reply:
Oh, misunderstanding.  I put the change back.